arm-servicebus/2014-09-01/swagger/servicebus.json

```diff
@@ -566,153 +572,2046 @@
+          "type": "string",
+          "format": "time-span",
+          "description": "TimeSpan structure that defines the duration of the duplicate detection history. The default value is 10 minutes.."
+        },
+        "EnableBatchedOperations": {
+          "type": "boolean",
+          "description": "Value that indicates whether server-side batched operations are enabled.."
+        },
+        "EnableExpress": {
+          "type": "boolean",
+          "description": "Value that indicates whether Express Entities are enabled. An express topic holds a message in memory temporarily before writing it to persistent storage."
+        },
+        "EnableFilteringMessagesBeforePublishing": {
+          "type": "boolean",
+          "description": "Whether messages should be filtered before publishing."
+        },
+        "EnablePartitioning": {
+          "type": "boolean",
+          "description": "Value that indicates whether the topic to be partitioned across multiple message brokers is enabled."
+        },
+        "ExtensionData": {
+          "type": "object",
+          "description": "Structure that contains extra data."
+        },
+        "IsAnonymousAccessible": {
+          "type": "boolean",
+          "description": "Value that indicates whether the message is anonymous accessible."
+        },
+        "IsReadOnly": {
+          "type": "boolean",
+          "description": "Value that indicates whether the entity description is read-only."
+        },
+        "MaxSizeInMegabytes ": {
+          "format": "int64",
+          "type": "integer",
+          "description": "Maximum size of the topic in megabytes, which is the size of memory allocated for the topic."
+        },
+        "MessageCountDetails": {
+          "type": "object",
+          "description": "Message details for a topic."
+        },
+        "Path": {
+          "type": "string",
+          "description": "Name of the topic."
+        },
+        "RequiresDuplicateDetection": {
+          "type": "boolean",
+          "description": "Value indicating if this topic requires duplicate detection."
+        },
+        "SizeInBytes ": {
+          "format": "int64",
+          "type": "integer",
+          "description": "Size of the topic in bytes."
+        },
+        "EntityStatus ": {
+          "type": "string",
+          "description": "Enumerates the possible values for the status of a messaging entity.",
+          "enum": [
+            "Active",
+            "Creating",
+            "Deleting",
+            "Disabled",
+            "ReceiveDisabled",
+            "Renaming",
+            "Restoring",
+            "SendDisabled",
+            "Unknown"
+          ],
+          "x-ms-enum": {
+            "name": "EntityStatus",
+            "modelAsString": false
+          }
+        },
+        "SubscriptionCount": {
+          "format": "int32",
+          "type": "integer",
+          "description": "Number of subscriptions."
```

Comment:
In general we do not want to have constraints on the client, we want them to be determined at the service side